packages/dataviews/src/add-filter.js

```diff
@@ -115,19 +115,12 @@ export default function AddFilter( { filters, view, onChangeView } ) {
 												activeElement?.value ===
 												element.value;
 											return (
-												<DropdownMenuItem
+												<DropdownMenuRadioItemCustom
 													key={ element.value }
-													role="menuitemradio"
-													aria-checked={ isActive }
-													prefix={
-														isActive && (
-															<Icon
-																icon={ check }
-															/>
-														)
-													}
-													onSelect={ ( event ) => {
-														event.preventDefault();
+													name={ `add-filter-${ filter.field.id }` }
```

Comment:
This name renders as `add-filter-undefined` right now. I'm fixing it at https://github.com/WordPress/gutenberg/pull/57416

According to [the att docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#name), it's expected that `name` is the same for each radio item, though wanted to check in case I've missed something.